packages/botonic-nlp/src/tasks/ner/process/processor.ts

```diff
@@ -25,7 +25,10 @@ export class Processor {
 
   // Generates the Input data and the unmasked sequence.
   generateInput(text: string): { sequence: string[]; input: InputData } {
-    const sequence = this.processText(text)
+    const sequence = this.preprocessor.pad(
```

Comment:
in tests/helpers/tasks/ner/helper.ts please rename MAX_LENGTH to st more descriptive . MAX_TOKEN_LENGTH?